src/webview.ts

```diff
@@ -0,0 +1,90 @@
+import * as vscode from "vscode";
+import * as path from "path";
+
+export function startWebview(context: vscode.ExtensionContext) {
+  // Create panel
+  const panel = vscode.window.createWebviewPanel(
+    "snootyPreview",
+    "Snooty Preview",
+    vscode.ViewColumn.Beside,
+    {
+      enableScripts: true,
+      retainContextWhenHidden: true
+    }
+  ); // Set up resource files needed for page to render (bundle and css) // If file does not exist, panel will show empty content // Paths of bundle and css files may change after snooty frontend submodule works
+
+  const bundlePath = path.join(
+    context.extensionPath,
+    "snooty-preview/preview",
+    "bundle.js"
+  );
+  const bundleFile = getResource(bundlePath); // TODO: Add validation through parser to see which css file to use. // For testing purposes, only support guides until we get communication between extension and frontend working
+  const cssPath = path.join(
+    context.extensionPath,
+    "snooty-preview/static/docs-tools",
+    "guides.css"
```

Comment:
Not always going to be guides.css

Reply:
This was left in for testing but will be changed in a separate PR to allow for multiple templates  ([DOCSP-7186](https://jira.mongodb.org/browse/DOCSP-7186)). Will make that PR after merging the PR for the parser